docs/sources/developers/plugins/legacy/defaults-and-editor-mode.md

```diff
@@ -59,7 +56,7 @@ If you want your users to be able to change these panel values then you need to
 
 ## Editor Mode
 
-Editor mode is when a user clicks Edit on a panel. Every panel has a general tab where you change the title and width and some panels have more inbuilt tabs like the Metrics tab or Time Range tab. A panel plugin can add its own tab(s) so that a user can customize the panel. 
+Editor mode is when a user clicks Edit on a panel. Every panel has a general tab where you change the title and width and some panels have more inbuilt tabs like the Metrics tab or Time Range tab. A panel plugin can add its own tab(s) so that a user can customize the panel.
 
 Grafana conventions mean all you need to do is to hook up an Angular template with input fields and Grafana will automatically save the values to the dashboard json and load them on dashboard load.
```

Comment:
What is going on here?

Reply:
Not sure. GitHub having a hiccup? Either way, it's not introduced in this PR, so I'm inclined to leave it as is for now.